playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -5,7 +5,7 @@ deployment_rhel7_ent_base:
   # rhel-7.1, requires cloud access subscription
   image: "{{ lookup('oo_option', 'ec2_image') | default('ami-10251c7a', True) }}"
   image_name: "{{ lookup('oo_option', 'ec2_image_name') | default(None, True) }}"
-  region: "{{ lookup('oo_option', 'AWS_REGION') | default('us-east-1', True) }}"
```

Comment:
Yes, I did mean to revert this. When testing this with ansible tower, it became apparent why they were using ec2-region.